boards/arm/arduino_due/arduino_due.dts

```diff
@@ -56,3 +56,5 @@
 &pa8a_uart_urxd {
 	bias-pull-up;
 };
+
+arduino_i2c: &twi1 { };
```

Comment:
LGTM! Maybe it is important make sure twister will test this config by adding a filter like this: https://github.com/zephyrproject-rtos/zephyr/blob/53ce87a9721d5172146f1f6d41eec55bf910ff4d/boards/arm/sam_v71_xult/sam_v71_xult.yaml#L13